warehouse/models/mart/gtfs/_mart_gtfs_dims.yml

```diff
@@ -143,6 +143,8 @@ models:
         description: |
           Flags rows with a duplicate _gtfs_key; e.g. area_id duplicates would result in _gtfs_key duplicates within a feed version.
           Treat these rows with caution as they can cause fanout when joining on natural (i.e. GTFS) keys.
+          This test excludes the regional MTC feed because it's a large feed that often violates GTFS uniqueness
```

Comment:
massive nit: should this be a comment not part of the column description? Don't think that including it in description will make sense once it's displayed in dbt docs